### PR TITLE
Update Colour.java

### LINKED ISSUE #219: Minor Request: Additional Human Skin Colours
Colour Name	RGB Colour Code
	Rosy		#DDA993
	Porcelain	#DBCDB9
	Tan		#C39D6B	
	Fake Tan	#B67D38
	Chocolate	#59372D

Could rename 'Fake Tan' as 'Dark Tan'.

### LINKED ISSUE #649: Duplicate color concerning the horn color in the demon transformation menu
**Version** : 0.2.5.8
**Date** : 26 May 2018

grey and dark-grey are  the same grey

Screenshot

### PR DESCRIPTION
**Pull request by Rfpnj**
Adds four additional tones for Human and Demon Skins.

### Things to include in a large Pull Request:

- What is the purpose of the pull request?
 resolves #219 requesting additional skins colors, resolves #649 horn color duplication
- Give a brief description of what you changed or added.
Of the five requested, I added four of them Porcelain, Rosy, Tanned, Chocolate. I skipped Fake tan, I felt it did not add any value to the choices.
- Are any new graphical assets required?
No
- Has this change been tested? If so, mention the version number that the test was based on.
0.2.5.5
- So we have a better idea of who you are, what is your Discord Handle?

- If you want quick feedback, you can use @Innoxia, or jump on the Lilith's Throne discord and send Innoxia a PM.

### CODE REVIEW

File: src/com/lilithsthrone/utils/Colour.java [hunk 1]
```diff
@@ -338,9 +338,13 @@
 	// Body parts:
 
 	// Skin (Human and Demon):
+	SKIN_PORCELAIN(Util.newColour(0xDBCDB9), Util.newColour(0xDBCDB9), "porcelain")
 	SKIN_PALE(Util.newColour(0xFBF4E9), Util.newColour(0x534946), "pale"),
 	SKIN_LIGHT(BaseColour.YELLOW_LIGHT, "light"),
+	SKIN_ROSY(Util.newColour(0xDDAA93), Util.newColour(0xDDAA93), "rosy")
```
Comment (Pimvgd) [inline src/com/lilithsthrone/utils/Colour.java]: missing comma at end

File: src/com/lilithsthrone/utils/Colour.java [hunk 2]
```diff
@@ -338,9 +338,13 @@
...
+	SKIN_TANNED(Util.newColour(0xC39D6B), Util.newColour(0xC39D6B), "Tanned")
```
Comment (Pimvgd) [inline src/com/lilithsthrone/utils/Colour.java]: and here

File: src/com/lilithsthrone/utils/Colour.java [hunk 3]
```diff
@@ -338,9 +338,13 @@
...
 	SKIN_OLIVE(BaseColour.TAN, "olive"),
+	SKIN_CHOCOLATE(Util.newColour(0x59372D), Util.newColour(0x59372D), "chocolate")
```
Comment (Pimvgd) [inline src/com/lilithsthrone/utils/Colour.java]: and here

File: src/com/lilithsthrone/utils/Colour.java [hunk 4]
```diff
@@ -338,9 +338,13 @@
 	// Body parts:
 
 	// Skin (Human and Demon):
+	SKIN_PORCELAIN(Util.newColour(0xDBCDB9), Util.newColour(0xDBCDB9), "porcelain")
```
Comment (Pimvgd) [inline src/com/lilithsthrone/utils/Colour.java]: this missing a comma too (so I guess all the rows in this section that you added)

File: src/com/lilithsthrone/utils/Colour.java [hunk 5]
```diff
@@ -513,62 +517,77 @@
...
+
+			new ListValue<Colour>(Colour.SKIN_PORCELAIN),
+			new ListValue<Colour>(Colour.SKIN_PALE),
+			new ListValue<Colour>(Colour.SKIN_LIGHT),
+			new ListValue<Colour>(Colour.SKIN_ROSY),
+			new ListValue<Colour>(Colour.SKIN_TANNED),
+			new ListValue<Colour>(Colour.SKIN_OLIVE),
+			new ListValue<Colour>(Colour.SKIN_CHOCOLATE),
+			new ListValue<Colour>(Colour.SKIN_DARK),
+			new ListValue<Colour>(Colour.SKIN_EBONY));
```
Comment (CognitiveMist) [inline src/com/lilithsthrone/utils/Colour.java]: It looks like Util.newArrayListOfValues() was adjusted to not require new ListValue<>()'s -- see ratSkinColous. This change should take that into account.

**Comment by Rfpnj** [comment]
And Done

**Comment by Innoxia** [comment]
Thank you! ^^

**Pull request merged**